meerk40t/core/units.py

```diff
@@ -528,47 +611,43 @@ def angle_preferred(self):
 
     @property
     def radians(self):
-        amount = self.angle
```

Comment:
The properties now give the non-rounded amounts whether or not digits are set. Digits only affect the formatting.